### ISSUE TITLE
Medusa downloads both from Usenet and Bittorrent

### ISSUE THREAD

**Issue by plato79** [pymedusa/Medusa#5915]
**Describe the bug**
When doing a backlog search for an episode Medusa sent the link to both usenet client and torrent client.

**To Reproduce**
Go to a show, select an episode and set it to wanted.

**Expected behavior**
It should download either from usenet or torrent

**Screenshots**

**Medusa (please complete the following information):**
 - OS: FreeNas 11.2
 - Branch: master
 - Commit: [918cfe7]

**Logs:**
<details>

```
2018-12-25 14:36:35 INFO     SEARCHQUEUE-BACKLOG-331753 :: [918cfe7] Building internal name cache for Black Clover
2018-12-25 14:36:35 INFO     SEARCHQUEUE-BACKLOG-331753 :: [918cfe7] Using backlog search providers
2018-12-25 14:36:35 INFO     SEARCHQUEUE-BACKLOG-331753 :: [Anizb] :: [918cfe7] Performing episode search for Black Clover
2018-12-25 14:36:35 INFO     SEARCHQUEUE-BACKLOG-331753 :: [Anizb] :: [918cfe7] Quality of [HorribleSubs].Black.Clover.-.58.[1080p] is 1080p HDTV
2018-12-25 14:36:35 INFO     SEARCHQUEUE-BACKLOG-331753 :: [Anizb] :: [918cfe7] Quality of [HorribleSubs].Black.Clover.-.58.[720p] is 720p HDTV
2018-12-25 14:36:35 INFO     SEARCHQUEUE-BACKLOG-331753 :: [Anizb] :: [918cfe7] Quality of [HorribleSubs].Black.Clover.-.58.[480p] is SDTV
2018-12-25 14:36:35 INFO     SEARCHQUEUE-BACKLOG-331753 :: [Anizb] :: [918cfe7] Picked [HorribleSubs].Black.Clover.-.58.[1080p] as the best result.
2018-12-25 14:36:35 INFO     SEARCHQUEUE-BACKLOG-331753 :: [HorribleSubs] :: [918cfe7] Performing episode search for Black Clover
2018-12-25 14:36:37 INFO     SEARCHQUEUE-BACKLOG-331753 :: [Rarbg] :: [918cfe7] Performing episode search for Black Clover
2018-12-25 14:36:49 INFO     SEARCHQUEUE-BACKLOG-331753 :: [Rarbg] :: [918cfe7] Performing season pack search for Black Clover
2018-12-25 14:36:54 INFO     SEARCHQUEUE-BACKLOG-331753 :: [Nyaa] :: [918cfe7] Performing episode search for Black Clover
2018-12-25 14:36:54 INFO     SEARCHQUEUE-BACKLOG-331753 :: [Nyaa] :: [918cfe7] Quality of [HorribleSubs].Black.Clover.-.58.[720p].mkv is 720p HDTV
2018-12-25 14:36:54 INFO     SEARCHQUEUE-BACKLOG-331753 :: [Nyaa] :: [918cfe7] Quality of [HorribleSubs].Black.Clover.-.58.[1080p].mkv is 1080p HDTV
2018-12-25 14:36:54 INFO     SEARCHQUEUE-BACKLOG-331753 :: [Nyaa] :: [918cfe7] Quality of [HorribleSubs].Black.Clover.-.58.[480p].mkv is SDTV
2018-12-25 14:36:54 INFO     SEARCHQUEUE-BACKLOG-331753 :: [Nyaa] :: [918cfe7] Picked [HorribleSubs].Black.Clover.-.58.[1080p].mkv as the best result.
2018-12-25 14:36:54 INFO     SEARCHQUEUE-BACKLOG-331753 :: [TokyoToshokan] :: [918cfe7] Performing episode search for Black Clover
2018-12-25 14:36:54 INFO     SEARCHQUEUE-BACKLOG-331753 :: [TokyoToshokan] :: [918cfe7] Quality of [HorribleSubs].Black.Clover.-.58.[720p].mkv is 720p HDTV
2018-12-25 14:36:54 INFO     SEARCHQUEUE-BACKLOG-331753 :: [TokyoToshokan] :: [918cfe7] Quality of [HorribleSubs].Black.Clover.-.58.[1080p].mkv is 1080p HDTV
2018-12-25 14:36:54 INFO     SEARCHQUEUE-BACKLOG-331753 :: [TokyoToshokan] :: [918cfe7] Quality of [HorribleSubs].Black.Clover.-.58.[480p].mkv is SDTV
2018-12-25 14:36:54 INFO     SEARCHQUEUE-BACKLOG-331753 :: [TokyoToshokan] :: [918cfe7] Picked [HorribleSubs].Black.Clover.-.58.[1080p].mkv as the best result.
2018-12-25 14:36:54 INFO     SEARCHQUEUE-BACKLOG-331753 :: [ThePirateBay] :: [918cfe7] Performing episode search for Black Clover
2018-12-25 14:36:56 INFO     SEARCHQUEUE-BACKLOG-331753 :: [LimeTorrents] :: [918cfe7] Performing episode search for Black Clover
2018-12-25 14:36:58 INFO     SEARCHQUEUE-BACKLOG-331753 :: [918cfe7] Downloading [HorribleSubs].Black.Clover.-.58.[1080p] with size: 1.24 GB from Anizb
2018-12-25 14:37:00 INFO     SEARCHQUEUE-BACKLOG-331753 :: [918cfe7] Downloading [HorribleSubs].Black.Clover.-.58.[1080p].mkv with 396 seeders and 6 leechers and size 1.10 GB from Nyaa
2018-12-25 14:37:01 INFO     SNATCHQUEUE-SNATCH-331753 :: [918cfe7] Beginning to snatch release: [HorribleSubs].Black.Clover.-.58.[1080p]
2018-12-25 14:37:01 INFO     SNATCHQUEUE-SNATCH-331753 :: [918cfe7] Downloading [HorribleSubs].Black.Clover.-.58.[1080p] with size: 1.24 GB from Anizb, through a SearchType.BACKLOG_SEARCH search
2018-12-25 14:37:01 INFO     SNATCHQUEUE-SNATCH-331753 :: [918cfe7] Sending NZB to SABnzbd
2018-12-25 14:37:01 INFO     SNATCHQUEUE-SNATCH-331753 :: [918cfe7] Sending slack message: Started Download: Black Clover - 058 - The Battlefield Decision - 1080p HDTV from Anizb
2018-12-25 14:37:01 INFO     SNATCHQUEUE-SNATCH-331753 :: [918cfe7] Sending slack message  to url: **********
2018-12-25 14:37:02 INFO     SEARCHQUEUE-BACKLOG-331753 :: [918cfe7] Downloading [HorribleSubs].Black.Clover.-.58.[1080p].mkv with 351 seeders and 6 leechers and size 0.00 B from TokyoToshokan
2018-12-25 14:37:04 INFO     SNATCHQUEUE-SNATCH-331753 :: [918cfe7] Beginning to snatch release: [HorribleSubs].Black.Clover.-.58.[1080p].mkv
2018-12-25 14:37:04 INFO     SNATCHQUEUE-SNATCH-331753 :: [918cfe7] Downloading [HorribleSubs].Black.Clover.-.58.[1080p].mkv with 351 seeders and 6 leechers and size 0.00 B from TokyoToshokan, through a SearchType.BACKLOG_SEARCH search
2018-12-25 14:37:05 INFO     SNATCHQUEUE-SNATCH-331753 :: [918cfe7] Sending slack message: Started Download: Black Clover - 058 - The Battlefield Decision - 1080p HDTV with 351 seeders and 6 leechers from TokyoToshokan
2018-12-25 14:37:05 INFO     SNATCHQUEUE-SNATCH-331753 :: [918cfe7] Sending slack message  to url: **********
```
</details>

**Comment by p0psicles** [comment]
Never mind, I reproduced it. I'll fix it asap through a hotfix.

**Comment by p0psicles** [comment]
Fixed in develop #5920
